x/mongo/driver/ocsp/ocsp.go

```diff
@@ -28,9 +27,6 @@ import (
 var (
 	tlsFeatureExtensionOID = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 1, 24}
 	mustStapleFeatureValue = big.NewInt(5)
-
-	defaultRequestTimeout = 5 * time.Second
```

Comment:
For `defaultRequestTimeout`, I wanted to move the timeout setting into `contactResponders` to reduce the possibility that it would get out-of-sync with the function comment that describes the timeout.

For `errGotOCSPResponse`, it's not needed anymore since there's no special logic for different errors returned by the `errgroup` goroutines.